src/abi/interaction.ts

```diff
@@ -21,7 +21,7 @@ interface ISmartContractWithinInteraction {
 }
 
 /**
- * Legacy component. Use "SmartContractTransactionsFactory" (for transactions) or "SmartContractQueriesController" (for queries), instead.
+ * @deprecated component. Use "SmartContractTransactionsFactory" (for transactions) or "SmartContractQueriesController" (for queries), instead.
```

Comment:
this does not exist anymore: `SmartContractQueriesController`. Change to:
```
Use "SmartContractTransactionsFactory" or "SmartContractController", instead.
```

or something similar.